akka-docs-dev/rst/scala/http/routing-dsl/directives/security-directives/index.rst

```diff
@@ -0,0 +1,35 @@
+.. _SecurityDirectives:
+
+SecurityDirectives
+==================
+
+.. toctree::
+   :maxdepth: 1
+
+   authenticate
+   authenticatePF
+   authenticateStrict
+   authenticateStrictPF
+   authenticateOrRejectWithChallenge
+   authorize
+   extractCredentials
+
+
+Authentication vs. Authorization
+--------------------------------
+
+*Authentication* is the process of establishing a known identity for the user, whereby 'identity' is defined in the
```

Comment:
Explain the role of HTTP in this picture (as in, the methods listed here map to HTTP, but there is no general purpose AAA solution provided here)